web/src/components/molecules/View/ViewDeletionModal.tsx

```diff
@@ -0,0 +1,42 @@
+import Alert from "@reearth-cms/components/atoms/Alert";
+import Button from "@reearth-cms/components/atoms/Button";
+import Modal from "@reearth-cms/components/atoms/Modal";
+import { useT } from "@reearth-cms/i18n";
+
+import { View } from "./types";
+
+export interface Props {
+  view?: View;
+  open: boolean;
+  onClose: (refetch?: boolean) => void;
```

Comment:
Is refetch used anywhere in the code?
```suggestion
  onClose: () => void;
```